extensions/binding/pom.xml

```diff
@@ -25,6 +25,7 @@
     <module>org.eclipse.smarthome.binding.ntp</module>
     <module>org.eclipse.smarthome.binding.ntp.test</module>
     <module>org.eclipse.smarthome.binding.sonos</module>
+    <module>org.eclipse.smarthome.binding.weatherunderground</module>
```

Comment:
Could you please also add the binding to the [p2 feature](https://github.com/eclipse/smarthome/blob/master/features/org.eclipse.smarthome.feature.runtime.binding/feature.xml) and as well to the [Karaf feature](https://github.com/eclipse/smarthome/blob/master/features/karaf/esh-ext/src/main/feature/feature.xml)?